core/src/components/divider/divider-vars.scss

```diff
@@ -1,14 +1,14 @@
 $border-types: top, right, bottom, left;
```

Comment:
```suggestion

```
It seems it is not used, so maybe just delete it?

Reply:
Sure, not really part of PR, but will fix!